packages/firestore/src/core/query.ts

```diff
@@ -43,7 +43,7 @@ export const enum LimitType {
 /**
  * The Query interface defines all external properties of a query.
  *
- * QueryImpl implements this interface to provide memoization for `queryOrderBy`
+ * QueryImpl implements this interface to provide memoization for `queryNormalizedOrderBy`
```

Comment:
+1

Thanks for making me look up "memoization" btw.

Reply:
I think the PR was misleading because it only showed you the first line. The sentence ends on the next line with "and \`queryToTarget\`." This includes proper punctuation, but the concern is noted.